parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh

```diff
@@ -10,6 +14,38 @@ removeContainerd() {
     apt_get_purge 10 5 300 moby-containerd
 }
 
+removeCurl() {
+    apt_get_purge 10 5 300 curl
+}
+
+installLatestCurlManuallyIfNotPresent() {
+    # Too Risky to manage within the components.json file since curl is required for the bootstrap logic to work
+    # currently this logic is only needed for 18.04, we will be able to remove this logic once we no longer support 18.04
+    
+    # renovate: datasource=github-releases depName=curl/curl
+    version="8.9.0"
+
+    if snap list curl > /dev/null; then
+        echo "curl is already installed and at the right version, skipping manual installation"
+        return
+    fi
+    
+    deb_file="/tmp/curl.tgz"
+    removeCurl || exit $ERR_CURL_REMOVE_TIMEOUT
+    retrycmd_if_failure 3 5 5 wget https://curl.haxx.se/download/curl-${version}.tar.gz -O $deb_file || exit $ERR_CURL_DOWNLOAD_TIMEOUT
```

Comment:
didn't we decide on keeping NC around for 1804 to avoid this?

Reply:
refresh, I removed this, once I realized I forgot.    Curl works when properly authenticated, no need for netcat